styleguide/components/21-atoms/headings/heading-2/_heading-2.scss

```diff
@@ -3,7 +3,7 @@ h2,
 %h2 {
   @include theme('color', 'color-tertiary', 'heading-2-color');
 
-  margin-bottom: 2.4rem;
+  margin-bottom: 2rem;
```

Comment:
Why rem here, and in the other files ems?

Reply:
@Jeroen005 All margin bottoms for all heading types use rem...